### PR TITLE
Team invites UX for authed user

### PR DESCRIPTION
**Pull request by sean-brydon**
Improves the UX for authed users to notify them that they have a pending invite.

Empty state when self hosted.
CleanShot 2023-01-24 at 12 00 49@2x

Empty state when hosted on cal.com
CleanShot 2023-01-24 at 12 01 34@2x

Pending invite 
CleanShot 2023-01-24 at 12 02 29@2x

Pending Invite and in a team 
CleanShot 2023-01-24 at 12 03 00@2x

### CODE REVIEW

File: packages/features/ee/teams/components/TeamListItem.tsx [hunk 1]
```diff
@@ -140,13 +141,14 @@ export default function TeamListItem(props: Props) {
...
-              <div className="hidden sm:block">
+              <div className="hidden justify-center sm:flex">
                 <Button type="button" color="secondary" onClick={declineInvite}>
                   {t("reject")}
                 </Button>
                 <Button
                   type="button"
-                  color="primary"
+                  color="secondary"
+                  StartIcon={FiCheck}
```
Comment (sean-brydon) [inline packages/features/ee/teams/components/TeamListItem.tsx:151]: Lets not have more than one primary action - gets a bit confusing

File: packages/features/tips/UpgradeTip.tsx [hunk 2]
```diff
@@ -63,23 +56,15 @@ export function UpgradeTip({
...
-                <p className="text-gray-700">{feature.description}</p>
-              </div>
-            ))}
+          {features.map((feature) => (
+            <div key={feature.title} className="mb-4 min-h-[180px] w-full rounded-md bg-gray-50 p-8 md:mb-0">
+              {feature.icon}
+              <h2 className="font-cal mt-4 text-lg">{feature.title}</h2>
+              <p className="text-gray-700">{feature.description}</p>
+            </div>
+          ))}
```
Comment (sean-brydon) [inline packages/features/tips/UpgradeTip.tsx:67]: Handle invites elsewhere now - cleaner and doesnt mess up with component design

**Review by PeerRich — APPROVED** [review]

File: packages/features/shell/TeamInviteBadge.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { useTeamInvites } from "@calcom/lib/hooks/useHasTeamPlan";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Badge } from "@calcom/ui";
+
+export function TeamInviteBadge() {
+  const { isLoading, listInvites } = useTeamInvites();
+  const { t } = useLocale();
+
+  if (isLoading || !listInvites || listInvites.length === 0) return null;
```
Comment (emrysal) [inline packages/features/shell/TeamInviteBadge.tsx:9]:
```suggestion
  if (isLoading || !listInvites?.length) return null;
```

**Review by emrysal — APPROVED** [review]

**Comment by Jaibles** [comment]
Great work @sean-brydon

**Pull request merged**